catalog/tests/dags/common/storage/test_media.py

```diff
@@ -709,6 +542,60 @@ def test_MediaStore_validates_filetype(filetype, url, expected_filetype):
     assert cleaned_data["filetype"] == expected_filetype
 
 
+# Extracts `jpg` extension from the url.
+# Converts `jpeg` to `jpg` to use the standard extension.
+# Extracts `tif` extension, and converts it to the standard, `tiff`.
+# Does not use extracted extension if it's not a valid image extension.
+# Uses the `filetype` even if the `url` extension is different.
```

Comment:
I _think_ these comments might have been moved erroneously (they seem to refer to the above test).

Reply:
Oops, weird. I don't remember moving this.